WireDomain/Tests/WireDomainTests/Helpers/TestSetup.swift

```diff
@@ -16,25 +16,22 @@
 // along with this program. If not, see http://www.gnu.org/licenses/.
 //
 
-import Foundation
+import WireTransport
+import WireTransportSupport
+import XCTest
 
-public class TestUserDefaults: UserDefaults {
-    private let suiteName: String
+final class TestSetup: NSObject, XCTestObservation {
```

Comment:
When/where does this get initialized?

Reply:
This is `initialized` by magic as it is configured to be the `NSPrincipalClass` in the bundle's info.plist. So it is the system itself that instantiates it.